client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -138,33 +141,6 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 		return null;
 	}
 
-	function getContent() {
-		return (
-			<>
-				<div className="confirm__info-section" />
-				<div className="confirm__instructions-container">
-					{ getWPComSubdomainWarningContent() }
-					{ getCheckoutContent() }
-					{ getWarningsOrHoldsSection() }
-					<ActionSection>
-						<SupportLink />
-						<StyledNextButton
-							disabled={ hasBlockers || ! isDataReady }
-							onClick={ () => {
-								if ( siteUpgrading.required ) {
-									return ( window.location.href = siteUpgrading.checkoutUrl );
-								}
-								goToStep( 'transfer' );
-							} }
-						>
-							{ __( 'Sounds good' ) }
-						</StyledNextButton>
-					</ActionSection>
-				</div>
-			</>
-		);
-	}
-
```

Comment:
Moved due to TS complaining about `dispatch` typing

Reply:
🤔